src/styles/blocks/variables.scss

```diff
@@ -0,0 +1,8 @@
+$card_width: 200px;
```

Comment:
In previous comments I meant that variables, extends files better put in `utils` folder and name from `_`)